### ISSUE TITLE
upper left + sign in circle is clickable, nothing happens - should go to add/create projects page

### ISSUE THREAD

**Issue by foundscapes** [OpenArchive/Save-app-android#201]

**Comment by n8fr8** [comment]
Can I get more detail? screenshot or notes? I can't reproduce this.

**Comment by foundscapes** [comment]
It's hard to show this as it's a tapping/clicking action.

Basically it indicates I'm clicking it when I touch the + in circle in upper left, but it doesn't take me to the add/choose project screen, nothing happens. Will test on new build!

**Comment by roseregina** [comment]
0.2.2-RC-1

MotoG and GalaxyS7

When I click on the circle plus symbol on a project view, I am taken to the "Add a Project" screen. 

@foundscapes can you test it again on the Pixel3A since that is where we initially had the issue?

**Comment by foundscapes** [comment]
Phone: Pixel 3a
OS: Android 10

This is still not working using the most updated version on play today. Nothing happens when I click the + in circle in upper left.


![Screenshot_20200323-134813](https://user-images.githubusercontent.com/2231711/77362406-19eaa800-6d0e-11ea-9c2a-f5510bdf5b5d.png)

**Comment by roseregina** [comment]
I've reproduced this now on the MotoG: 

Expected results: 
1. Log in to server
2. Click circle plus in upper left
3. Goes to "Add a Project" screen 
4. Add project

Actual results: 
1. Log in to server
2. Click circle plus in upper left
**3. Nothing happens**
4. Click on "Add a Project" text in center of screen
5. Goes to "Add a Project" screen 
6. Add project
7. Return to server view
8. Click on circle plus in upper left
9. Goes to "Add a Project" screen  *works as expected*
4. Add project


Basically, the circle plus in the upper left does not work until after the first project is added. At a minimum, this means that it needs to appear in grey until after the first project is added, but ideally, it would work consistently, especially since there is already a little bit of inconsistency with if the teal text is clickable.

**Comment by foundscapes** [comment]
Thank you @roseregina!

Can we just make it always clickable to add project @n8fr8 since that is what the expected interaction is?

**Comment by foundscapes** [comment]
Save version: 0.2.3-BETA-1
phone: pixel 3a
OS: android 10

This is working now on pixel 3a, please test on other phones @roseregina w/ 0.2.3-BETA-1 version

**Comment by roseregina** [comment]
Save version: 0.2.3-BETA-1
phone: GalaxyS7
OS: android 8.0.0 w/ Samsung experience 9.0 

Circle-plus is clickable as described above

**Comment by foundscapes** [comment]
awesome!
